src/init.ts

```diff
@@ -21,7 +21,19 @@ import * as path from 'path';
 import {Options} from './cli';
 import {readFilep as read, readJsonp as readJson, writeFileAtomicp as write} from './util';
 
-const pkg = require('../../package.json') as PackageJson;
+const pkg = require('../../package.json');
```

Comment:
Why did the type annotation here disappear?

Reply:
That wasn't an annotation but rather a cast. The cast was not necessary.

Reply:
Also – we don't use this yet – but going forward we would stronger types for package.json available from new resolveJsonModule in TS 2.9.